tdrs-backend/tdpservice/parsers/schema_defs/tanf/t4.py

```diff
@@ -20,27 +20,30 @@
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.dateMonthIsValid()]),
               Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isAlphaNumeric(),
```

Comment:
@elipe17 here too, i think we'd like it to be consistent with the validators in section 1. whichever is most complete. 

```
Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
                        required=True, validators=[
                              validators.isAlphaNumeric(),
                              validators.notMatches('_'*11),
                              validators.notEmpty(),
                        ]),
```